app/controllers/application_controller.rb

```diff
@@ -4,27 +4,30 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   protect_from_forgery with: :null_session, :if => Proc.new { |c| c.request.format == 'application/json' }
 
-  # for session tokens
-  # acts_as_token_authentication_handler_for User
-
   before_action :configure_permitted_parameters, if: :devise_controller?
 
-
-  # Custom Error Methos
+  # Custom Error Methods
+  # These are app level errors that we can use to configure how to handle them
+  # They should be created when there is not an already applicable Rails error
+  # Use 
   module SnapException
     class AccessDenied < StandardError; end
-  end 
+  end
 
   # Clean Error Handler Methods
   # Use these in place of handling errors individually.
-  def access_denied # Equivalent 401
-    raise SnapException::AccessDenied.new('You don\'t have permission to view this.')
+  def access_denied(msg= nil)  # Equivalent 401
```

Comment:
Unnecessary spacing detected.<br>Surrounding space missing in default value assignment.